packages/equinoxe/equinoxe.0.1.0/opam

```diff
@@ -0,0 +1,44 @@
+opam-version: "2.0"
+synopsis: "An OCaml wrapper for the Equinix API"
+description: "Equinoxe is an OCaml wrapper for the Equinix API."
+maintainer: ["Étienne Marais <etienne@maiste.fr>"]
+authors: ["Étienne Marais <etienne@maiste.fr>"]
+license: "MIT"
+homepage: "https://github.com/maiste/equinoxe"
+doc: "maiste.github.io/equinoxe"
+bug-reports: "https://github.com/maiste/equinoxe/issues"
+depends: [
+  "dune" {>= "2.9"}
+  "ocaml" {>= "4.08.0"}
+  "odoc" {with-doc}
+  "ezjsonm" {>= "1.3.0"}
+  "alcotest" {with-test}
+  "cmdliner"
+  "piaf"
+]
+build: [
+  ["dune" "subst"] {dev}
+  [
+    "dune"
+    "build"
+    "-p"
+    name
+    "-j"
+    jobs
+    "--promote-install-files=false"
+    "@install"
+    "@runtest" {with-test}
```

Comment:
```suggestion
#    "@runtest" {with-test}
```
Since tests cannot run in the CI (sandbox)

Reply:
I've updated the tests so they are self-contained. It will be better for my test suite! Thanks for the suggestion :)

Reply:
even better!